### PR TITLE
Add GetItem converter

### PR DESCRIPTION
**Pull request by disktnk**
- support np.newaxis
- support Ellipsis (`...`)
- not support step slicing (`[0:999:2]`)
- not support advanced indexing (`array[[0,1],[0,1]]`)

### CODE REVIEW

File: onnx_chainer/functions/array.py [hunk 1]
```diff
@@ -52,10 +52,85 @@ def convert_Depth2Space(func, onnx_op_name, opset_version, input_names,
...
+            ends.append(idx+1)
+            squeeze_idxs.append(axis)
+        elif idx is None:
+            unsqueeze_idxs.append(i - len(squeeze_idxs) + skipped)
+        elif idx is Ellipsis:
+            # calculate rest slice number except None, GetItem does not allow
+            # multiple Ellipsis, so ignore latter Ellipsis count
+            rest_slice_len = len(
+                [idx_ for idx_ in func.slices[i+1:] if idx_ is not None])
+            for ellipsis_i in range(axis, len(x.shape)-rest_slice_len):
```
Comment (shinh) [inline onnx_chainer/functions/array.py]: I'm not sure, but is this loop actually necessary?

```
assert skiped == 0
skipped = len(x.shape) - rest_slice_len - 1
```
or something would work?

File: onnx_chainer/functions/array.py [hunk 2]
```diff
@@ -52,10 +52,85 @@ def convert_Depth2Space(func, onnx_op_name, opset_version, input_names,
...
+                axes.append(ellipsis_i)
+                starts.append(0)
+                ends.append(x.shape[ellipsis_i])
+                skipped += 1
+            skipped -= 1  # for next loop
+        else:
+            # not support advanced index like `array[[0,1], [0, 1]]`
+            raise ValueError(
+                'GetItem with type {} cannot handle in ONNX Slice, so that '
```
Comment (shinh) [inline onnx_chainer/functions/array.py]: I guess this rejects `chainer.Variable(np.array([1,2,3]))[np.array(1)]`, which can be implemented by ONNX's `Slice`? I'm not sure if it's worth supporting though..

File: onnx_chainer/functions/array.py [hunk 3]
```diff
@@ -52,10 +52,85 @@ def convert_Depth2Space(func, onnx_op_name, opset_version, input_names,
...
+            axes=squeeze_idxs))
+
+    nodes.append(helper.make_node(
+        onnx_op_name, input_names, slice_out_names,
+        axes=axes, starts=starts, ends=ends))
+
+    return tuple(nodes)
+
+
+dummy_objects = []
```
Comment (shinh) [inline onnx_chainer/functions/array.py]: Wow, I didn't realize onnx-chainer has this weird hack. I hope we eventually re-implement the default naming scheme not to use `id`

**Comment by disktnk** [comment]
By failing to fix conflict with master branch, commit history is awkward, I replace another PR.

**Pull request closed without merging**